EG23.tex

```diff
@@ -147,7 +147,9 @@ \subsection{Reactors}
 After the LWR to SFR transition is completed, new SFR reactors are deployed
 with a lower breeding ratio in order to prevent unbounded buildup of
 plutonium. The properties of the different reactors cores used for the
-simulation have been summarized in Table \ref{tab:reactor}.
+simulation have been summarized in Table \ref{tab:reactor}. As some liberty have
```

Comment:
don't like the phrase "liberty have been taken"  It sounds too imprecise and not careful